test/services/user-group/hooks/classes.test.js

```diff
@@ -11,7 +11,7 @@ const {
 	restrictToUsersOwnClasses,
 } = require('../../../../src/services/user-group/hooks/classes');
 
-describe('class hooks', () => {
+describe.only('class hooks', () => {
```

Comment:
remove the only please.
You can run a test or test class with this command "cross-env NODE_ENV=test mocha -g "your test describe which you want to run" --exit --timeout 15000"